mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -70,6 +77,8 @@ CLASS net/minecraft/class_140 net/minecraft/entity/Entity
 		ARG 1 tag
 	METHOD method_350 deserializeDefaultData (Lnet/minecraft/class_5;)V
 		ARG 1 tag
+	METHOD method_352 initPlayer ()V
+	METHOD method_353 setDead ()V
```

Comment:
Influenced by MCP, I'd presume? Maybe something more informative could work, possibly like this? 
```suggestion
	METHOD method_353 markForRemoval ()V
```